src/components/Zap/BackButton/BackButton.tsx

```diff
@@ -0,0 +1,35 @@
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import { Typography } from '@mui/material';
+import Link from 'next/link';
+import { useTranslation } from 'react-i18next';
+import { BackButtonStyles } from './BackButton.style';
+
+interface BackButtonProps {
+  path?: string;
+  title?: string;
+}
+
+export const BackButton = ({ path, title }: BackButtonProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Link
+      href={path ?? '/profile'}
+      style={{ textDecoration: 'none', width: 168, display: 'block' }}
+    >
+      <BackButtonStyles size={'large'}>
+        <ArrowBackIcon
+          sx={{ color: '#FFFFFF', width: '16px', height: '16px' }}
+        />
+        <Typography
+          fontSize="14px"
+          lineHeight="14px"
+          fontWeight={500}
+          marginLeft={'8px'}
+        >
+          {String(title || t('navbar.navbarMenu.profile')).toUpperCase()}
```

Comment:
Not 100% sure about it but if the translation does not exists, it will be null or undefined, toUpperCase() will then crash the app

Reply:
Good point! Not sure if it´s a good manner, but I did it like so now: 'Jumper Profile' as last resort!

`String(title || t('navbar.navbarMenu.profile') || 'Jumper Profile',).toUpperCase()}`